### PR TITLE
[Excalibur] Population modal updates

### LINKED ISSUE #349: [Excalibur] UI Updates for the population modal
**What needs to be done? Why does it need to be done?**

- ~Update the copy on the two modal pages~
- ~Add the tooltips next to the form labels~
- ~Change default starting date to 2/1/2020~
- ~add baselinePopulations property to duplicate scenario function~
- ~Edit: When only 1 facility is modelled, update copy for the title and body (2 instances) to say "1 facility" instead of "1 facility"~
image

https://www.figma.com/file/6z1QQW84a59Ul3c17PgWdV/%F0%9F%9A%80Recidiviz-CJStatus-(excalibur)?node-id=1837%3A494

**Additional context**
Updated mocks:
image

image

Tooltip text:
- Date: "To model the impact since the start of COVID-19, enter a date prior to any COVID-19 mitigation efforts."
- Total incarcerated population: "Enter the total incarcerated population across all modelled facilities at the selected date."
- Total staff: "Enter the total staff population across all modelled facilities at the selected date."

### PR DESCRIPTION
**Pull request by daschi**
## Description of the change

- Updates main copy on both pages of the population modal 
- Adds tooltips next to the form labels
- Adds `baselinePopulations` to the `duplicateScenario` process (cc / @sprad please let me know if this looks correct)

## Type of change

- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

Closes #349 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing has been performed locally

### Code review

These boxes should be checked by reviewers prior to merging:

- [x] This pull request has a descriptive title and information useful to a reviewer
- [x] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [x] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

**Comment by jovergaag** [comment]
@daschi This looks good to me. I'll let @sprad chime in on the `duplicateScenario` process before approving.

**Comment by daschi** [comment]
@jessex Thanks for the suggestion, I just tried it out and was able to duplicate the scenario from the scenario library and it copied all of the baseline populations with no issues. cc @jovergaag

File: src/page-response-impact/BaselinePopulationModal/index.tsx [hunk 1]
```diff
@@ -58,12 +57,20 @@ const Subtitle = styled.span`
   }
 `;
 
-const ModalTitle: React.FC<Pick<Props, "numFacilities">> = ({
+const ModalTitle: React.FC<Pick<Props, "numFacilities"> & { page: number }> = ({
```
Comment (rocktacular) [inline src/page-response-impact/BaselinePopulationModal/index.tsx:60]: cool, had to look up this TS syntax. Should we explicitly define a prop type for this component for clarity?

Comment (daschi) [reply src/page-response-impact/BaselinePopulationModal/index.tsx:60]: Sure, I can pull this definition out and assign it to a type

**Pull request merged**